docs/internal/admin/List-Core-Actions.md

```diff
@@ -0,0 +1,288 @@
+# List of Core Actions
+
+The snapshot loaded with the system at the start called `core_templates` has a list of all core-actions included in template `Sample template`.
+
+## Disclaimer
+
+The plugin actions that will be listed here are different from actions that users can act in the sytem, this are only meant for admins and configure how the system will work.
+Core-actions aren't meant to be modified, and only someone with expertise should be dealing with these. Otherwise some of the features that rely on certain actions would stop working.
+
+## Briefing
+
+Core-actions are listed in each template and are configurations that run after certain events happen triggered by changes done by users in Applications linked to the template.
+These actions should be added at the start of a template creation - one siimple way is to duplicate the `Sample template` to start with all of these core-actions in the new template configuration.
+Some of the plugin actions - which aren't core-actions - are also defined per template (and some examples at the end) used mainly for configuring notifications, generatioin of certificates and outcomes.
+
+## Lists
+
+This list should reflect the same as we have in `Sample templates` so when changing that we need to also add to this list.
+
+### Templates
+
+- Edit User Details (No review)
+- User Registration (No review)
+- Company Registration (1 stage)
+  - Review - permission: `reviewGeneral`
+- Join Company (1 stage)
+  - Review: - permissioin: `reviewGeneral`
+- Demo -- Features Showcase (3 stages)
+  - Screening - permission: `reviewScreening` self-assignable
+  - Assemessment: (2 levels)
+    - 1. Review permissions: `reviewGeneral` and assigner: `assignGeneral`
+    - 2. Consolidation permission: `reviewAssessmentLevel2` self-assignable
```

Comment:
I thought we were going to call this "reviewSenior" or something like that?

Reply:
This is just to keep the list of permissions short. But for now I would say let's merge like this and change later if desirable (remembering to update here as well)